### ISSUE TITLE
Develop title bar UI 

### ISSUE THREAD

**Issue by kengoy** [sfbrigade/stop-covid19-sfbayarea#107]
This is a ticket to develop the title bar. (light purple part with the title text)

Question about UI wireframe, style: ask @justinjiangrws 
Question about current Dev: ask @kengoy 

<img width="319" alt="Screen Shot 2020-05-06 at 9 40 17 PM" src="https://user-images.githubusercontent.com/61864097/81255332-40c11d00-8fe2-11ea-8496-b0dd8f5b1fcf.png">
<img width="738" alt="Screen Shot 2020-05-06 at 9 40 07 PM" src="https://user-images.githubusercontent.com/61864097/81255336-428ae080-8fe2-11ea-87e8-336e6bdfc3d9.png">

**Comment by kengoy** [comment]
Latest UI wireframe: https://www.figma.com/file/iUS5LdAnuTKVei88QwPPCH/Bay-Area-Pandemic-Dashboard-(PanDa)-by-Bay-Area-Brigades?node-id=602%3A127

**Comment by ilakyapal** [comment]
@kengoy Is someone actively working on this issue or is it available to help with?

**Comment by kengoy** [comment]
@ilakyapal Yes, this task is being in progress. Let me think another issue for you.
